tsdb/chunkenc/histo.go

```diff
@@ -247,7 +248,17 @@ func (a *HistoAppender) Append(int64, float64) {}
 // * the zerobucket threshold has changed
 // * any buckets disappeared
 // * there was a counter reset in the count of observations or in any bucket, including the zero bucket
+// * the last sample in the chunk was stale while the current sample is not stale
```

Comment:
Sorry for commenting yet again, but I think we should add that this method will always return `true` if the current sample is stale.